### PR TITLE
Update About Pages

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
This is a content and style update for the About page. There should be no functional changes in this PR.

The changes intended in this PR include:
- The About page needs to look like this: [InVision Preview](https://projects.invisionapp.com/share/Y6DLOTUNE#/screens/254604074_SotCG-About) (requires auth access)
  - with style reference available here: [InVision Inspect mode](https://projects.invisionapp.com/d/main#/console/10495701/254604074/inspect) 
- The About page needs to have the following text: [About Page Doc](https://docs.google.com/document/d/1YckwDXbWd_NlvwHLGo4AxHcWsIBw3ZHHlTRqbpLhEUA/edit#) (requires auth access)
  - The text needs to be readable in English, Hebrew, and Arabic.

### Status
WIP

### CODE REVIEW

**Comment by shaunanoordin** [comment]
Quick checkpoint update:
- Intro, done
- About the Geniza, done
- About the Research partners, in progress, need to register image assets
- Image Partners, not started, need to register image assets
- Provenance, not started
- The Team + Special Thanks, no changes 
- Arabic, Hebrew: not started. I think it makes more sense to split WHOLE SECTIONS of the About page to, e.g. `section_partners_ar.jsx` instead of doing `translate(string)` for each paragraph of text since it'll be a nightmare to manage, and we'd have to faff about with inline HTML like <a> links.

**Comment by shaunanoordin** [comment]
## PR Update
The About page now has (near) full content, styling, and functionality.
- Styling follows Becky's designs, with as few exceptions as possible.
- Styling accounts for mobile design and RtL design.
- About page now has content for English, Hebrew, and Arabic. 
- Some bits of text aren't translated, mind you.

Dev notes:
- Each section has its own _translated component,_ instead of having sections with _translated strings._ This was due to the unwieldy amount of text that the Translate system would need to handle (i.e. one string for _each paragraph)_ as well as the nightmare that would have been the inline HTML (links, etc)
- The RtL switching is done via _CSS styles,_ since 1. I wanted to keep the HTML to a minimum and 2. I didn't want to create individual React Components for each flippable component (there were so, so many floats). This decision assumes that the About page will be a mostly-static information page, as opposed to something that'll be updated in any frequent manner - while the _content_ (HTML) is much easier to read, the price is that the CSS is trickier to understand.

### Status
OK, we've got English, Hebrew, and Arabic - this PR is now ready for review @wgranger

**Comment by wgranger** [comment]
Note: Raha should be removed from the team page.

File: src/components/about/Index.jsx [hunk 1]
```diff
@@ -3,10 +3,32 @@ import { getTranslate, getActiveLanguage } from 'react-localize-redux';
...
+import ContentIntroHe from './content-intro-he.jsx';
+import ContentIntroAr from './content-intro-ar.jsx';
+
+import ContentGenizaEn from './content-geniza-en.jsx';
+import ContentGenizaHe from './content-geniza-he.jsx';
+import ContentGenizaAr from './content-geniza-ar.jsx';
+
+import ContentResearchEn from './content-research-en.jsx';
+import ContentResearchHe from './content-research-he.jsx';
+import ContentResearchAr from './content-research-ar.jsx';
```
Comment (wgranger) [inline src/components/about/Index.jsx]: Were these imported in such a way, as opposed to inserting them into locales, for styling reasons?

File: src/components/about/content-geniza-en.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      <p className="text">Beginning in the Renaissance, European diplomats, scholars, and travelers to the Middle East and North Africa generated a vibrant antiquities market as they actively sought and purchased texts and artifacts to bring home with them for study or to add to their personal libraries. During the 19th century, the contents of the Cairo Geniza began to draw attention when assorted fragments began to come up for sale. Orientalists, traders, collectors, and scholars amassed these fragments for their own collections. Today, Cairo Geniza fragments can be found in collections from throughout the world. To learn about the history of the collections the fragments you are transcribing come from, <a href="https://docs.google.com/document/d/1uxY_EgX9qUp-oQMFbFboZvMs3t-nd9FLLibhVO37xdA/edit?usp=sharing" target="_blank">click here</a> or scroll down.</p>
```
Comment (wgranger) [inline src/components/about/content-geniza-en.jsx]: This link isn't in the Arabic translations. Also, should this be a Google doc?

**Comment by wgranger** [comment]
This code looks very well organized and there are only a few comments which need to be addressed above. One final suggestion is that we go ahead and insert the social media links for the buttons. I know this was not an initial concern of this PR, but if we have them handy it would be easy to slip in now so we don't forget (since the links appear on the about page).

File: src/components/about/Index.jsx (hunk 1 above)
Comment (shaunanoordin) [reply src/components/about/Index.jsx]: I decided to package whole sections into components to make the content easier to read and review (and contrast between different languages), and so the stylings could be managed much more easily. "Easy to read" is my personal opinion though; it's not too difficult for me to change the code to use `/locales/your-language-here.js` if we deem it more suitable for long-term code maintenance.

File: src/components/about/content-geniza-en.jsx (hunk 2 above)
Comment (shaunanoordin) [reply src/components/about/content-geniza-en.jsx]: I have _several questions_ about the translations, and yes, this is one of them.
- Check out [the translations document](https://docs.google.com/document/d/1YckwDXbWd_NlvwHLGo4AxHcWsIBw3ZHHlTRqbpLhEUA/edit#) (Google Docs, Zooniverse login required)
- The English version links to Google Docs, and actually describes details similar to the Provenance section.
- The Hebrew version helpfully marks where the last link should appear... but only has 3 paragraphs to contrast the 5 paragraphs of English
- The Arabic version has no markings to indicate where links should appear.

The differences are pretty stark once you compare the `content-geniza-LANG.js` files side by side. 🤷‍♂️

**Comment by shaunanoordin** [comment]
## PR update
- Raha removed from the team page.
- Links added to social media.
- Unfortunately, I can't do much to fix the Hebrew/Arabic translations for the "About the Geniza" section, but I'm open to suggestions.

File: src/components/about/Index.jsx (hunk 1 above)
Comment (wgranger) [reply src/components/about/Index.jsx]: I think we should remove this as it seems awkward to link to a Google document on our page. @snblickhan - do you agree with this? The about section contains the following sentence in English and Hebrew "To learn about the history of the collections the fragments you are transcribing come from, [click here](https://docs.google.com/document/d/1uxY_EgX9qUp-oQMFbFboZvMs3t-nd9FLLibhVO37xdA/edit) or scroll down."

File: src/components/about/content-geniza-en.jsx (hunk 2 above)
Comment (wgranger) [reply src/components/about/content-geniza-en.jsx]: I think we should remove this as it seems awkward to link to a Google document on our page. @snblickhan - do you agree with this? The about section contains the following sentence with link in English and Hebrew "To learn about the history of the collections the fragments you are transcribing come from, [click here](https://docs.google.com/document/d/1uxY_EgX9qUp-oQMFbFboZvMs3t-nd9FLLibhVO37xdA/edit) or scroll down."

Comment (snblickhan) [reply src/components/about/content-geniza-en.jsx]: We don't want to link to any Google docs anywhere.

The "click here" text was included in the event that we had links to sub-sections within the About document -- it should link to the Provenance section of the About page.

**Comment by shaunanoordin** [comment]
## PR Update
- Google Docs link has been removed and replaced with a local link to the `#provenance` section.
- PR conflicts fixed by merging master to this branch.

**Comment by wgranger** [comment]
Great. Thanks for the update. Merging.

**Pull request merged**